Formula/llvm.rb

```diff
@@ -381,8 +402,21 @@ def caveats
               "-I#{MacOS.sdk_path}/usr/include",
               "-L#{lib}",
               "-Wl,-rpath,#{lib}", "test.cpp", "-o", "test"
-      assert_includes MachO::Tools.dylibs("test"), "#{opt_lib}/libc++.1.dylib"
+      assert_includes MachO::Tools.dylibs("test"), "@rpath/libc++.1.dylib"
```

Comment:
Well, the tests are failing (see https://bot.brew.sh/job/Homebrew%20Core/17783/), the question is for original author of this test, because when you build llvm from this formula it sets installation_path for the libraries as @rpath. You can use `otool -l [path to libc++ library] to see that LC_ID_DYLIB is set to `@rpath`. 

It is possible that LLVM have changed it from being absolute by default to relative, but I am surprised how it got updated in homebrew-core without seeing breakage in the tests. And I could not find any flags in libcxx project which can change this behavior.